### DISCUSSION TITLE
What are internal plugins?

### DISCUSSION THREAD

**Discussion by otakutyrant** [neovim/neovim#24048] [Q&A] [answered]
`newtr` is an internal plugin for Vim at least. However I cannot find out what NeoVim's internal plugins are. Btw I found some `Pi_*` in the document. What does `Pi_` stand for?

**Comment by clason** [comment, marked as answer]
"Internal plugins" are plugins bundled with the Neovim runtime: https://github.com/neovim/neovim/tree/master/runtime/plugin

As to what `pi` stands for; probably "plugin internal", if I had to guess, as these are the help files for bundled plugins. I wouldn't put too much meaning into it, though.
